components/tikv_util/src/metrics/mod.rs

```diff
@@ -36,16 +46,63 @@ mod metrics_reader;
 
 pub type RecordPairVec = Vec<pdpb::RecordPair>;
 
-pub fn dump(should_simplify: bool) -> String {
+lazy_static! {
+    // the registry for high priority metrics.
+    pub static ref HIGH_PRIORITY_REGISTRY: Registry = Registry::new();
+}
+
+static METRICS_COMPACT_LEVZEL: AtomicU32 = AtomicU32::new(MetricsCompactLevel::NoCompact as u32);
+static METRICS_LEVEL: AtomicU32 = AtomicU32::new(MetricsLevel::All as u32);
+
+// reduce the return frequency of normal metrics factor
+const NORMAL_METRICS_REDUCE_FACTOR: u64 = 2;
```

Comment:
Can the factor changed?

Reply:
As most of the function aggregation window size are 60s(some are 30s), thus, if the factor is bigger than 2, the charts in grafana will output abnormal output. A factor of 3 will result in abnormal output because some window contains 2 sample and others are only 1. Facotr >=4 will make most charts empty(in this case we can manually increase the aggregation window size.)  So, I'm not sure if there are scenario that this factor can be more than 2.